runelite-client/src/main/java/net/runelite/client/plugins/corp/CorpDamageOverlay.java

```diff
@@ -82,8 +83,11 @@ public Dimension render(Graphics2D graphics)
 		int totalDamage = corpPlugin.getTotalDamage();
 		int players = corpPlugin.getPlayers().size();
 
-		// estimate how much damage is required for kill based on number of players
-		int damageForKill = players != 0 ? totalDamage / players : 0;
+		// estimate the probability of your damage exceeding everyone else's
+		double mean = players != 0 ? totalDamage / players : 0;
+		double sd = Math.sqrt(totalDamage / (0.1 * players));
+		double probabilityHigherThanOne = new NormalDistribution(mean, sd).cumulativeProbability(myDamage);
```

Comment:
Library-less cdf approximation you are free to use: https://gist.github.com/iProdigy/f5cbc9fff300f674790fab8dbac9e46e